src/linear_codes/mod.rs

```diff
@@ -93,18 +93,21 @@ where
     /// of a polynomial of degree m - 1.
     fn encode(msg: &[F], param: &Self::LinCodePCParams) -> Vec<F>;
 
-    /// Get the representation of the polynomial
-    fn poly_repr(polynomial: &P) -> Vec<F>;
+    /// Represent the polynomial as either coefficients,
+    /// in the univariate case, or evaluations over
+    /// the Boolean hypercube, in the multilinear case.
+    fn poly_to_vec(polynomial: &P) -> Vec<F>;
 
-    /// How we choose to split the query point into a Vec of Field elements.
-    /// Needed for appending to transcript.
+    /// Represent the query point as a vector of Field elements.
     fn point_to_vec(point: P::Point) -> Vec<F>;
 
-    /// Compute the matrices for the polynomial
+    /// Arrange the coefficients of the polynomial into a matrix,
+    /// and apply encoding to each row.
+    /// Returns the tuple (original_matrix, encoded_matrix).
     fn compute_matrices(polynomial: &P, param: &Self::LinCodePCParams) -> (Matrix<F>, Matrix<F>) {
-        let mut coeffs = Self::poly_repr(polynomial);
+        let mut coeffs = Self::poly_to_vec(polynomial);
 
-        // 1. Computing parameters and initial matrix
+        // 1. Computing the matrix dimensions.
         let (n_rows, n_cols) = param.compute_dimensions(coeffs.len()); // TODO for 6 coefficients, this is returning 4 x 2 with a row of 0s: fix
```

Comment:
Should this TODO still be here? Has it been fixed? And if not, do we want to push with it inside?